src/man/include/debug_levels_tools.xml

```diff
@@ -58,6 +58,11 @@
          <emphasis>9</emphasis>,
          <emphasis>0x4000</emphasis>: Extremely low-level tracing information.
     </para>
+    <para>
+         <emphasis>10</emphasis>,
+         <emphasis>0x10000</emphasis>: Even more low-level libldb tracing
```

Comment:
If this log level supposed to be used only for libldb tracking? Rest of the debug levels seems to be more "general", only this one sticks out.

Reply:
Yes, as I wrote in the commit message, this new debug level is introduced especially for LDB_DEBUG_TRACE.

The idea is that users are used to "debug_level=9" and I want LDB_DEBUG_TRACE messages to go away from those logs.

I am open for suggestions how to handle this better.

Reply:
Well, users are used to level 9 because this is what developers suggests. Bitmask levels are in for 9 years and AFAIK I'm the only one who's always asking for 0x3ff0.

The original idea behind bitmask was to have the ability to disable certain messages (such as ldb). Number is certainly more user friendly but as long as we have direct map between number and bitmask combination we will never actually use this.

Reply:
> Well, users are used to level 9 because this is what developers suggests. Bitmask levels are in for 9 years and AFAIK I'm the only one who's always asking for 0x3ff0.

I agree masks provide more fine grained customization.

But in absence of own level for LDB messages mask doesn't help to exclude it: we really often want to get SSSDBG_TRACE_ALL to debug a problem.